src/components/ui/TripCardAccordion/AccordionDetails/components/DetailsGrid.tsx

```diff
@@ -5,36 +5,53 @@ import { BusDepartureIcon } from '../../../../../assets/accordianDetails/BusDepa
 import { BusArrivalIcon } from '../../../../../assets/accordianDetails/BusArrivalIcon';
 import { StopwatchIcon } from '../../../../../assets/accordianDetails/StopwatchIcon';
 import { BusIcon } from '../../../../../assets/accordianDetails/BusIcon';
+import { ISeatType, IBusType, ILocation } from '../../../../../types';
 
-export const DetailsGrid = () => {
+export interface IDetailsGridProps {
+    departureTime: string;
+    departureDate: string;
+    arrivalTime: string;
+    arrivalDate: string;
+    duration: string;
+    origin: ILocation;
+    destination: ILocation;
+    seatType: ISeatType;
+    busType: IBusType;
+}
+
+export const DetailsGrid = ({
+    tripDetails,
+}: {
+    tripDetails: IDetailsGridProps;
+}) => {
     const { t } = useTranslation('tripDetails');
 
     const detailsData = [
         {
             title: t('departure'),
-            value: 'Trivandrum, 20th Jan, 5:00 am',
+            value: `${tripDetails.origin.name} ${tripDetails.departureDate}, ${tripDetails.departureTime}`,
```

Comment:
better to give a " , " after ${tripDetails.origin.name}

Reply:
Resolved in 9d923edd79563665e3e9d293787e963a7825f8ae